config/vufind/config.ini

```
@@ -2694,6 +2694,21 @@ description = "The REST API provides access to search functions and records cont
 ; URL pointing to a Terms of Service page (optional, default is none):
 ;termsOfServiceUrl = "https://something"
 
+; These settings control usage of API keys
+[API_Keys]
+; Mode for using API keys.
+; 'disabled' means API keys are not in use and cannot be created. Default.
+; 'enabled' allows users to create API keys and use them, but it is not mandatory.
+; 'enforced' forces users to create and provide an API key in a header field (see
+; header_field setting below).
+;mode = 'enabled'
+; Token salt to add when generating a new API key for a user.
+;token_salt =
+; Which header field key should the API key be provided. Default is X-API-KEY.
```

Clarifying language:

```suggestion
; The header field key that should be used to provide the API key. Default is X-API-KEY.
```